scripts/build/templates/index.js

```diff
@@ -1,3 +1,5 @@
+console.warn('Deprecation warning: The `simple-icons` entrypoint will be removed in the next major. Please switch to using `import * as icons from "simple-icons/icons"` if you need an object with all the icons.')
```

Comment:
This souldn't be inside the `Get.value` function?

Reply:
I think we may have different perceptions of what I intended to do in #6789. I meant to remove the whole file. If we remove the `.Get` function, the index file no longer has a use as you can get an object with all the icons using `import * as icons from 'simple-icons/icons'`

Reply:
Ah OK, I think I messed up a bit here 👍🏼

Reply:
But, could be a way to import with `import * as icons from 'simple-icons'`? Perhaps an alias in exports?

Reply:
Ah yes, that's a good idea. I'll do that.

Reply:
Yes, it seems simpler and more straightforward to me. That change would be in #6789 right?

Reply:
It's not breaking so I could do it here

Reply:
Oh also please do not release the major until my next PR. I'm still experimenting with one more change

Reply:
> It's not breaking so I could do it here

Great

> Oh also please do not release the major until my next PR. I'm still experimenting with one more change

It's scheduled in two days, so I think that you've enough time. It's done automatically like the rest of the releases. The [release-action](https://github.com/simple-icons/release-action) checks if there is at least one removal of an icon to do a major version bump.

Reply:
> But, could be a way to import with import * as icons from 'simple-icons'? Perhaps an alias in exports?

Looked into doing this and I don't think it'll work well tree-shaking and bundle size wise so I'm reluctant to do it until we remove what is being deprecated in this PR.